sass/components/_btn.scss

```diff
@@ -2,3 +2,10 @@
   line-height: 1;
   padding: .25rem 1rem;
 }
+
+.btn-primary:disabled,
+.btn-primary.is-disabled {
```

Comment:
the issue with that is that this already has some styles (namely the opacity) from basscss which we would need to override